reference/5.1/Microsoft.PowerShell.Utility/Out-File.md

```diff
@@ -356,15 +359,14 @@ You can pipe any object to `Out-File`.
 
 ## NOTES
 
-The `Out` cmdlets do not format objects; they just render them and send them to the specified
-display destination. If you send an unformatted object to an `Out` cmdlet, the cmdlet sends it to a
-formatting cmdlet before rendering it.
+If you send an unformatted object to an `Out` cmdlet, the cmdlet sends it to a formatting cmdlet
+before rendering it.
 
 To send a PowerShell command's output to the `Out-File` cmdlet, use the pipeline. You can store data
```

Comment:
Perhaps start the second sentence with "Alternatively, [you can store data ...]".

Reply:
Good clarifier. Fixed